features/admin.users.v1/components/user-change-password.tsx

```diff
@@ -629,7 +652,7 @@ export const ChangePasswordComponent: FunctionComponent<ChangePasswordPropsInter
      * configured in the server.
      */
     const resolveModalContent = () => {
-        if (governanceConnectorProperties?.length > 1) {
+        if (isResetPassword && governanceConnectorProperties?.length > 1) {
```

Comment:
is updating this condition intentional?

Reply:
Yes it was intentional as I during "Set a password" we concluded to show only the admin password setting option.